aspnetcore/tutorials/first-web-api/sample/src/TodoApi/Startup.cs

```diff
@@ -26,7 +26,11 @@ public Startup(IHostingEnvironment env)
         public void ConfigureServices(IServiceCollection services)
         {
             // Add framework services.
-            services.AddMvc();
+            services.AddMvcCore()
+                .AddAuthorization()
```

Comment:
Are all these services (that the sample is not using) recommended for minimal Web API apps?

cc @danroth27